packages/f-searchbox/src/components/formElements/FormSearchField.vue

```diff
@@ -22,42 +22,106 @@
                     [$style['c-search-input']]: true,
                     [$style['is-notEmpty']]: address
                 }"
-                @input="$emit('input', $event.target.value)">
+                @input="$emit('input', $event.target.value)"
+                @focus="toggleEnterLeaveInput(true)"
+                @blur="toggleEnterLeaveInput(false)">
 
             <span :class="$style['c-search-placeholder']">{{ copy.fieldPlaceholder }}</span>
         </label>
+
+        <form-search-inner-field-wrapper
+            :copy="copy"
+            :service="service" />
     </div>
 </template>
 
 <script>
+import { mapActions, mapState } from 'vuex';
+import FormSearchInnerFieldWrapper from './FormSearchInnerFieldWrapper.vue';
+
+const ALLOWED_SELECTION_TIME = 500;
+
 export default {
+    components: {
+        FormSearchInnerFieldWrapper
+    },
     props: {
         errorMessage: {
             type: [Boolean, String],
             default: false
         },
+
         address: {
             type: String,
             default: ''
         },
+
         isCompressed: {
             type: Boolean,
             default: true
         },
+
         copy: {
             type: Object,
             default:  () => ({})
         },
+
         shouldDisplayCustomAutocomplete: {
             type: Boolean,
             default: false
+        },
+
+        service: {
+            type: Object,
+            default: () => {},
```

Comment:
I don't think you need a default on a required prop

Reply:
Fixed.